### DISCUSSION TITLE
[目录显示] 文章使用Synced block，在博客里不会显示目录

### DISCUSSION THREAD

**Discussion by polarisaspire** [tangly1024/NotionNext#1966] [Q&A]
**描述bug**
Notion里的文章使用**Synced block**，在博客网页上不会显示目录，之前旧版本是可以的

**复现步骤**
Notion数据库里的文章采用 **Synced block**引入，在网页显示的时候，不展示目录导航；
不采用 **Synced block**，正常展示导航栏

**期望的正常结果**
网页显示的时候，展示目录导航

**截图**
![image](https://github.com/tangly1024/NotionNext/assets/16836821/8bd54821-bf2d-407f-b430-e32a3d25e75e)
![image](https://github.com/tangly1024/NotionNext/assets/16836821/119bee8e-ff5e-4ece-b6fa-a6ac05b4d6f5)
不采用**Synced block**正常显示
![image](https://github.com/tangly1024/NotionNext/assets/16836821/20d3cfb7-72b4-41fd-bcfc-dd1651d9a9d6)

**环境**

- 操作系统: windows
- 浏览器 : chrome
- NotionNext版本: 3.16.3
- 主题: 所有主题

**Comment by sloth2012** [comment]
这个怎么样了？除了这种，我试了下页面link跳转过去的notion文章也不显示目录
